src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -56,70 +61,160 @@ public static AbstractResponse create(String appName, String appVersion, int jre
 		return res;
 	}
 
+    /**
+     * Gets the app name
+     *
+     * @return The app name.
+     */
 	public String getAppName() {
 		return appName;
 	}
-
+    
+    /**
+     * Name of the IOTA software you're currently using (IRI stands for Initial Reference Implementation)
```

Comment:
Name and version no?
Especially if the `@return` won't be in the doclet.

Reply:
Doesn't IRI stand for IOTA reference implementation?
Reference: https://docs.iota.org/iri
> The IOTA Reference implimentation, commonly known as IRI, is the reference implimentation of the IOTA 'node' software in Java.
(Just realised the docs also have typo)

Reply:
I got the text from https://iota.readme.io/reference#getnodeinfo yea.
IOTA reference impl. makes much more sense. I will check and update accordingly